src/rdo.rs

```diff
@@ -34,6 +34,23 @@ use std;
 use std::vec::Vec;
 use crate::partition::PartitionType::*;
 
+#[derive(Copy,Clone)]
+pub enum RDOType {
+  Full,
```

Comment:
Yes, rephrasing the name so that we can understand from it will help us to use it correctly!

Reply:
Do you have a suggestion for names?

Reply:
Maybe, 
Pixel Domain + Full Rate ==> PixelDistRealRate?
TxDistRate = tx_dist plus tx_dist_rate ==> TxDistRealRate
MetricEst = SATD based rate and distortion estimate ==> SatdDistSatdRate
(And you can make them all cap and boldfaced!)

Reply:
Or, to save space,
RD_PIXEL_REAL
RD_TX_REAL
RD_SATD_SATD
?

Reply:
How's this?